locksmith/src/routes/v2/contracts.ts

```diff
@@ -0,0 +1,13 @@
+import express from 'express'
+import { ContractsController } from '../../controllers/v2/contractsController'
+import { authMiddleware } from '../../utils/middlewares/auth'
+
+const router = express.Router({ mergeParams: true })
+
+const contractsController = new ContractsController()
+
+router.post('/:network/lock', authMiddleware, (req, res) => {
```

Comment:
Or here rather: captcha middleware

Reply:
We already have Recaptcha logic somwhere in locksmith.

Reply:
Added